docs/api-docs/payments/payments-api-overview.md

```diff
@@ -197,7 +197,7 @@ lineNumbers: true
 **Get payment methods = process payment**
 * type = type
 * token = token
-* last_four = verification_value
+* CVC = verification_value
 * id = payment_method_id
 
```

Comment:
```suggestion
* id = payment_method_id

When paying with a stored card, the card's CVV can be sent in the `verification_value` field. If it is included, it will be sent to the payment provider wherever possible and used for verification. If you do not have the CVV then this field should be excluded. 
```